pandas/_libs/tslibs/period.pyx

```diff
@@ -1273,17 +1273,17 @@ cdef object _period_strftime(int64_t value, int freq, object fmt):
                 raise ValueError('Unable to get quarter and year')
 
             if i == 0:
-                repl = '%d' % quarter
+                repl = str(quarter)
             elif i == 1:  # %f, 2-digit year
-                repl = '%.2d' % (year % 100)
+                repl = f'{year % 100:02d}'
             elif i == 2:
-                repl = '%d' % year
+                repl = str(year)
             elif i == 3:
-                repl = '%03d' % (value % 1000)
+                repl = f'{value % 1_000:03d}'
             elif i == 4:
-                repl = '%06d' % (value % 1000000)
+                repl = f'{value % 1_000_000:06d}'
             elif i == 5:
-                repl = '%09d' % (value % 1000000000)
+                repl = f'{value % 1_000_000_000:09d}'
```

Comment:
are the underscores supported in py36?

Reply:
can you restore the parentheses around the %

Reply:
> are the underscores supported in py36?

Yes. see [pep 515](https://www.python.org/dev/peps/pep-0515/)